cpanfile

```diff
@@ -39,7 +39,7 @@ requires 'Cpanel::JSON::XS';
 requires 'JavaScript::Minifier::XS', '>= 0.11';
 requires 'JSON::Validator';
 requires 'LWP::UserAgent';
-requires 'Minion', '>= 10.0';
+requires 'Minion', '>= 10.04';
```

Comment:
What's the motivation for this? Is it required? Does it bring in features we'll want to make use of?

Reply:
Yes, it's required for one of the new Minion features used.